Varsha2001/Task1.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Google First</title>
+</head>
+<body>
+<h1 style="color: red;">First Task</h1>
+<p style="max-width: 400px;text-align:justify;">Hey welcome in Google First <br> How are you. </p>
+<div style="color:blue;"><p>Here is all HTML learning plateform for you.
+Learn the Basics of <b>HTML</b> from here.
+HTML is the standard markup language for Web pages.
+HTML describes the structure of a Web page
+HTML consists of a series of elements
+HTML elements tell the browser how to display the content
+HTML elements label pieces of content such as "this is a heading", "this is a paragraph", "this is a link", etc.</p></div>
+<p>Web pages can be created and modified by using professional HTML editors.
+
+However, for learning HTML we recommend a simple text editor like Notepad (PC) or TextEdit (Mac).
+
+We believe that using a simple text editor is a good way to learn HTML.</p>
+<hr>
+<p>Here is a song video linked with my page.</p>
+<hr>
+<h2>-:List:-</h2>
```

Comment:
This looks good however, you can remove this code as it is not required in task.